west.yml

```diff
@@ -97,7 +97,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: aeeb6a08afd0529c366728183367adf009104a4a
+      revision: pull/234/head
```

Comment:
b69c5a7c25007fbc111a0cbda6e2a6a945a6ae9a

Reply:
You probably meant to replace the revision [here](https://github.com/zephyrproject-rtos/zephyr/pull/27460/files#diff-4ae7a6d2c692e706a33f7a71d3eb25d2R59)

Reply:
I did, sorry.